src/server/scripts/EasternKingdoms/Karazhan/boss_nightbane.cpp

```diff
@@ -219,9 +219,7 @@ struct boss_nightbane : public BossAI
 
     void JustDied(Unit* /*killer*/) override
     {
-        if (instance)
-            instance->SetData(DATA_NIGHTBANE, DONE);
-
+        _JustDied();
```

Comment:
```suggestion
        BossAI::JustDied(killer);
```

I believe this is the format we have to keep as the methods are inherited from BossAI and are not private methods (even though they essentially do the same). Also needs to be done for AttackStart, MoveInLineOfSight and JustEngagedWith

Reply:
Trinitycore doesn’t follow this format (I remember being told the reason why but I don’t remember) therefore I am not changing this